views/js/pciCreator/dev/textReaderInteraction/runtime/css/textReaderInteraction.css

```diff
@@ -0,0 +1,163 @@
+.textReaderInteraction {
+    margin-top: 10px;
+}
+.tr-wrap {
```

Comment:
Please also scope all your css with a top class : .textRenderInteraction for instance.
This will prevent css rules clashing if another component happens to have the same css class as you